mix.exs

```diff
@@ -50,14 +51,16 @@ defmodule Spacesuit.Mixfile do
     [
       # All envs
       {:hackney, "~> 1.7.1", override: true},
-      {:cowboy, github: "extend/cowboy"},
+      {:cowboy, "~> 2.5", override: true},
```

Comment:
This is definitely not going to work without a bunch of code changes. This is written against an interim build of Cowboy between version 1 and 2.